### DISCUSSION TITLE
ZZZ Fps Unlock

### DISCUSSION THREAD

**Discussion by StressOzz** [34736384/genshin-fps-unlock#430] [Ideas]
Can we expect a version for Zenless Zone Zero ?
;)

**Comment by 34736384** [comment]
![image](https://github.com/34736384/genshin-fps-unlock/assets/25654009/3bb29ce8-72b4-4851-99ea-723f917470d4)

**Reply by Abagur900** [reply]
We need fps unlock for android

**Comment by Kawaiizilla** [comment]
This is still the FPS cap option, but the value is set to 300.

**Comment by ofecer007** [comment]
i have rtx3090 and have 70fps on 1080p and 64fps 4k =P

**Reply by Abagur900** [reply]
CPU bottleneck probably.
